apps/dashboard/app/views/projects/index.html.erb

```diff
@@ -38,17 +38,5 @@
         <% end %>
       </div>
 
-      <%- if @templates.size.positive? -%>
-      <div class="mt-5 justify-content-center text-center project-icon">
-        <%= link_to(new_project_path({template: true}),
-                      title: I18n.t('dashboard.jobs_create_template_project'),
-                      class: 'text-dark btn btn-link') do
-        %>
-          <%= icon_tag(URI.parse("fas://plus")) %><br>
-          <%= I18n.t('dashboard.jobs_create_template_project') %>
-        <% end %>
-      </div>
-      <%- end -%>
```

Comment:
I consciously made a separate button to distinguish the 2 workflows. I thought it was a better UX when the user _knew_ they were building from a template as apposed to clicking the new button project and wondering what templates where.

I have to admit I'm not a UX expert - I just thought the explicit difference would help beginners.

Reply:
Well it really was meant as a question - do you have some UX reason to change it?

I'm just sharpening my UX skills, so again, I'm not really sure - I just _thought_ 2 different flows would limit confusion.

Reply:
I would agree if the flows were actually different, and by selecting the template button the user would be sent to a  different set of pages. This might be the future state with a more complex template flow, but at the moment, it just adds the template dropdown that is why I thought it was simpler if merged.

Reply:
I see. Yea I don't anticipate it being more complex - but again it was a UX driven decision.

I'd say let's keep it the way it is assuming the way it is is a better user experience (which is my assumption, but I'd love if someone could confirm it).

Reply:
Updated the PR to keep the template flow as it was.